README.md

```diff
@@ -17,6 +17,14 @@ especially if you are using zopflipng for compression.  Intermediate products
 (compressed image files, for example) will be put into a build subdirectory; the
 font will be at the top level.
 
+## Using NotoColorEmoji
+
+NotoColorEmoji uses the CBDT/CBLC color font format, which is supported by Android
+and ChromeOS, but neither Windows nor MacOS.  No Browser on MacOS supports it, but
```

Comment:
Sorry that I forgot to note this. The official name is 'Google Chrome OS'. To be 'exact', it should be 'Google Chrome OS or Chromium OS'.